### PR TITLE
A few DM guide spelling fixes and a section on the room environment

### PR DESCRIPTION
**Pull request by pauldoomgov**
Removes some typos and grammatical mistakes.

Adds a section on supporting a positive learning environment.

### CODE REVIEW

File: _articles/wargames-dungeon-masters-guide.md [hunk 1]
```diff
@@ -174,4 +174,21 @@ Suggested flow:
...
+* Consider breaking into small rooms if you feel there are too many people to
+  maintain comfort
+* The DM can influence the atmosphere in the room - Try to keep it positive,
+  action oriented, and generative
+* Sometimes groups are in a quiet mood or not "feeling it", which should not
+  be seen as a failing by the DM
+* [Psychological safety](https://risk-engineering.org/concept/psychological-safety) 
+  is a prerequisite for effective incident response and learning so take immediate action
+  if you see signs of a room where players feel unable to openly share, question,
+  and risk being wrong
```
Comment (aduth) [inline _articles/wargames-dungeon-masters-guide.md:194]: Something about the "and learning so take" is making this sentence difficult for me to interpret.

Maybe clearer?

```suggestion
* [Psychological safety](https://risk-engineering.org/concept/psychological-safety) 
  is a prerequisite for effective incident response and learning to take place, so be sure to take immediate action
  if you see signs of a room where players feel unable to openly share, question,
  and risk being wrong
```

Here are some of my alternative (presumably wrong) interpretations:

```suggestion
* [Psychological safety](https://risk-engineering.org/concept/psychological-safety) 
  is a prerequisite for effective incident response, and learning to take immediate action
  is important if you see signs of a room where players feel unable to openly share, question,
  and risk being wrong
```

```suggestion
* [Psychological safety](https://risk-engineering.org/concept/psychological-safety) 
  is a prerequisite for effective incident response and, in learning so, take immediate action
  if you see signs of a room where players feel unable to openly share, question,
  and risk being wrong
```

**Review by aduth — APPROVED** [review]
LGTM

File: _articles/wargames-dungeon-masters-guide.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Contingency Plan Training Wargames - Dungeon Master's Guide"
-description: "How to create an exciting scenario"
+description: "How to create an exciting scenario as a Wargames DM"
```
Comment (jmdembe) [inline _articles/wargames-dungeon-masters-guide.md:3]: Let's flip the order here as Wargames DM is the big picture and scenarios are different kinds of Wargames

```suggestion
description: "How to create an exciting Wargames DM scenario"
```

File: _articles/wargames-dungeon-masters-guide.md [hunk 3]
```diff
@@ -9,10 +9,10 @@ subcategory: Guides
 ## Overview
 
 Login.gov's "Contingency Plan Training Wargames" are a regular event that use a
-mix of process practice, chaos engineering, and group troubleshooting to hon
+mix of process practice, chaos engineering, and group troubleshooting to hone
 skills and improve the rediness of our socio-technical system.
 
-The fuel for our Wargames are spine chilling scenarios created by our "Dungeon Masters".
+The fuel for our Wargames are spine tingling scenarios created by our "Dungeon Masters".
```
Comment (jmdembe) [inline _articles/wargames-dungeon-masters-guide.md:15]: Hyphenate spine-tingling

```suggestion
The fuel for our Wargames are spine-tingling scenarios created by our "Dungeon Masters".
```

File: _articles/wargames-dungeon-masters-guide.md [hunk 4]
```diff
@@ -132,9 +132,9 @@ types of fault, you may conduct the scenario the same as a [Classic Wargame](#cl
 ### What Was That?
 
 Looking at logs or metrics of production systems will often result in finding
-mysteries.  "Why did that line go up?"  "Is that and error or just noise?"
+mysteries.  "Why did that line go up?" or "Is that an error or is it just noise?"
 
-The lazy DM can crowd source and answer with a "What Was That?" session.
+A clever/lazy DM can crowd source an answer using a "What Was That?" session.
```
Comment (jmdembe) [inline _articles/wargames-dungeon-masters-guide.md:137]: Capitalization is unnecessary here

```suggestion
A clever/lazy DM can crowd source an answer using a "What was that?" session.
```

File: _articles/wargames-dungeon-masters-guide.md [hunk 5]
```diff
@@ -174,4 +174,21 @@ Suggested flow:
   it back
 * Complete the session with a full group discussion
 
-
+## Maintaining an Effective Environment
+
+* You may need to assign roles or ask for specific players to help if you do
+  not get volunteers
+* Remind players to share screens and talk through what they are doing and why
+  they are doing it
```
Comment (jmdembe) [inline _articles/wargames-dungeon-masters-guide.md:182]: Nit: what is "it"?

```suggestion
  they are doing it
```

**Comment by pauldoomgov** [comment]
Thanks for helping polish this @aduth and @jmdembe !

**Pull request merged**